cf/i18n/resources/fr_FR.all.json

```diff
@@ -5218,5 +5218,15 @@
       "id": "This service doesn't support creation of keys.",
       "translation": "Ce service ne supporte pas la création de clefs.",
       "modified": false
+   },
+   {
+      "id": "Space name (To see the apps in specified space of current organization)",
+      "translation": "Space name (To see the apps in specified space of current organization)",
```

Comment:
Here is the French translation: `"Nom de l'espace (pour voir les applications the l'espace specifié pour l'organisation courrante)"`